### ISSUE TITLE
Add accessibility keys like title to help screen readers

### ISSUE THREAD

**Issue by githiago-f** [githiago-f/e-coffee-frontend#19]
![image](https://user-images.githubusercontent.com/52945005/112191841-2e91a280-8be5-11eb-93fe-dec9afe59f50.png)
--------------------------------------
There are some focusable elements at this application that will be used by people with different vision levels, and they should have all of its accessibility keys set for helping them to read and interact with those elements.
